### ISSUE TITLE
Non default Column name causes get_primary_key to fail.

### ISSUE THREAD

**Issue by repole** [marshmallow-code/marshmallow-sqlalchemy#44]
Say you have a model:

```
class Author(Base):
    __tablename__ = 'Author'

    author_id = Column("AuthorId", Integer, primary_key=True)
    name = Column("Name", Unicode(160), nullable=False)
```

get_primary_key will return AuthorId as the primary key, but the attribute that's actually desired is author_id. When I go go to serialize a BookSchema (with a many relationship on authors), the list of Author primary keys incorrectly gets filled with None. Using the below snippet instead solves the issue for me:

```
from sqlalchemy.orm import ColumnProperty, class_mapper

def get_primary_columns(model):
    """Get primary key columns for a SQLAlchemy model.

    :param model: SQLAlchemy model class
    """
    primary_keys = []
    for primary_key in class_mapper(model).primary_key:
        for prop in class_mapper(model).iterate_properties:
            if isinstance(prop, ColumnProperty):
                if prop.columns:
                    if prop.columns[0].compare(primary_key):
                        primary_keys.append(prop)
                        break
    return primary_keys
```

Would love some input here on whether this all makes sense.

**Comment by jmcarp** [comment]
Makes sense to me--thanks for catching this. I proposed a slightly simpler fix in #45. It's basically the same as your solution, but a little shorter:

``` python
def get_primary_keys(model):
    """Get primary key properties for a SQLAlchemy model.

    :param model: SQLAlchemy model class
    """
    mapper = model.__mapper__
    return [
        mapper.get_property_by_column(column)
        for column in mapper.primary_key
    ]
```

Would you mind checking out the patch and verifying that it resolves the issue you encountered?

**Comment by repole** [comment]
Learn something new with SQLAlchemy every day, didn't realize get_property_by_column existed. Much more graceful solution.

Just tried it out and it works as expected. Thanks!
